rocky/reports/report_types/systems_report/report.py

```diff
@@ -126,8 +126,11 @@ def generate_data(self, input_ooi: str, valid_time: datetime) -> Dict[str, Any]:
         total_systems = len(ip_services)
         total_domains = 0
 
+        domains = []
```

Comment:
We can just create a set immediately instead of first creating a list and then converting it to a set to count (github doesn't let me create the suggestion in a single comment):

```suggestion
        domains = set()
```

Reply:
Good idea and suggestion. Thanks!